### PR TITLE
Fixing a few buttons to better fit rus translation

### PR DESCRIPTION
**Pull request by dth88**
Fixed few buttons to fit Russian translation, were too annoying.
Centered coin-wallet header data. much prettier I think, lmk.
Extended Russian Translation further, only Eula left it seems atm.

### CODE REVIEW

**Comment by smk762** [comment]
Perhaps @tonymorony might be better able to read for this one, my Russian is a bit недостаточный

**Comment by Milerius** [comment]
From what i've tested locally it's looks good in russian, but i'm not a native speaker, atleast the buttons are large enough

**Comment by dth88** [comment]
There might be few mistakes here and there bcz too much text and soapy eyes by the end. I can also translate the eula, decided to let it hang since probably proper lawyers should write that text, but anyway, will do it as well if my translation would be considered.

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:136]: `ПРЕДУПРЕЖДЕНИЕ: Приложение будет немедленно перезапущено для применения изменений!`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:227]: `адрес` word contains single `c`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:300]: `адрес` word contains single `c`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:310]: `адрес` word contains single `c`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:526]: `тся` writing without `ь`, initial translation was fine

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3240]: initial translation was correct

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1559]: there is no `нету` word in Russian

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:184]: above `теги` spelling used
https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/863/files#diff-bd6de22ea5ed42741c56c99ae64a1f6e314593b3a5dd17e6f105ae814a1f8840R174

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:420]: trailing whitespace

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1098]: initial translation was clearer

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1131]: использования

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1325]: in context it's more `Минимальный объем сделки`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1336]: `баланс менее чем размер комиссий`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1370]: Вы получите

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1375]: Вы отправите

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1385]: `Цена на CEX` is descriptive enough

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1380]: `фиатная цена`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1447]: `Discord` is not russian word

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1452]: `Twitter` is not russian word

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1457]: there is `руководство` translation in russian for `guides` word

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1465]: `не включен`
`возможность выбрать`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1509]: initial translation was correct

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1519]: initial translation was correct

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:1597]: there is no `order` word in initial label

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3066]: `за обычный транзакции`
`варироваться`
`Network fees can vary greatly depending on your selected trading pair.`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3083]: it was minified for purpose in initial translation - there is <comment> tag

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3089]: it was minified for purpose in initial translation - there is <comment> tag

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3120]: initial translation was correct

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3167]: Mode = `режим`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3172]: Mode = `режим`

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3195]: initial translation was fine

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3220]: initial translation was fine

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3225]: initial translation was fine

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3230]: initial translation was fine

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:3273]: initial translation was fine

File: atomic_defi_design/qml/Portfolio/Portfolio.qml
Comment (tonymorony) [inline atomic_defi_design/qml/Portfolio/Portfolio.qml:224]: if you want to propose GUI changes - please create separate PR to not mix it with translation

**Review by tonymorony — CHANGES_REQUESTED** [review]

File: atomic_defi_design/qml/Portfolio/Portfolio.qml
Comment (dth88) [reply atomic_defi_design/qml/Portfolio/Portfolio.qml:224]: there were just 2 or 3 buttons that needed a fix, and then I could not stop translating, you are right, will definitely keep in mind for future contributions though. however translation and button fixes lie in separate commits if this helps somehow.

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:3089]: sure if cut word is what you need here, np. But it does fit great in full form isn't it?

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:3083]: not sure what...  would you like me to do here? seems like this comment is for english translation and doesn't apply to Russian one. you want it to be cut like `фактич` ?

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:184]: sry, but I do not understand what you mean

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:420]: the problem here is that trailing whitespace is in the English source and this Linguist app for qt/qml demands that the translation should mirror the source.

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:1457]: could you please write the full sentence here that I need to write in translation tag, cant figure out how guides and `руководство` fits in together in this context...

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:420]: Screenshot 2021-04-27 at 15 12 24
turns out this is being done on purpose.

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts]: forgot...the problem here is that:
Screenshot 2021-04-27 at 16 09 01

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:184]: I meant that above ` <translation>Поиск контакта по имени или тегам</translation>` using `тЕги` spelling, and here `тЭги` - since it's english word I believe both spelling correct but it's better to use same in both places

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:3082]: yes, `фактич` cut is fine here, maybe worth to add the dot (`фактич.`)

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:3088]: same as `фактич` comment

File: atomic_defi_design/qml/Portfolio/Portfolio.qml
Comment (tonymorony) [reply atomic_defi_design/qml/Portfolio/Portfolio.qml:224]: as long as it's fine for @Milerius

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts]: changed it to "Форум и база знаний"

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts]: should I leave it as 'Открыть' or change to a more suitable translation 'Установить' ?

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:3082]: cant do 'фактич.'  qml-linguist app does not allow different endings. how about just 'факт'?

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (dth88) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:3088]: Still a bit unclear here... leaving `прибл` ? or maybe a more suitable `прогноз`?

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:2783]: yeah, in such context Set/Установить definetly more correct

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts
Comment (tonymorony) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts]: прибл / фактич is fine imo - there is a факт word so might be confusing without additional letters

**Comment by dth88** [comment]
Hopefully now everything is up to your standards guys. @Milerius @tonymorony @SirSevenG

**Review by Milerius — APPROVED** [review]

**Comment by Milerius** [comment]
I approved based on the QML changes, still waiting for @tonymorony approval for the merge !

**Review by tonymorony — APPROVED** [review]

**Pull request merged**
